src/components/moviepage/MoviePage.js

```diff
@@ -2,20 +2,29 @@ import React from 'react';
 import './MoviePage.css';
 
 const MoviePage = ({ details, viewHome }) => {
-  console.log('details', details[0].poster_path)
+  const { backdrop_path, title, genres, release_date, budget, revenue, runtime, average_rating, tagline, overview } = details[0].movie
+  let dollarUSLocale = Intl.NumberFormat('en-US');  
```

Comment:
great use of destructuring, and I love to see you formatting the dollars _before_ the return!